app/javascript/App/containers/coinSearch/sagas.js

```diff
@@ -10,7 +11,7 @@ function* searchCoins({ searchText, searchOpts, namespace }) {
   if (searchText.length < 2) return
   yield apiSagas.get(
     '/coins.json',
-    Object.assign({ q: { name_or_symbol_cont: searchText } }, searchOpts),
+    _.merge({ q: { name_or_symbol_cont: searchText } }, searchOpts),
```

Comment:
why did you choose to use `_.merge` instead of `Object.assign`?

Reply:
`Object.assign` is shallow